nanshe_ipython.ipynb

```diff
@@ -1356,29 +1365,28 @@
         "\n",
         "    imgs_min, imgs_max = 0, 100\n",
         "\n",
-        "    with open_zarr(fn, \"r\") as f:\n",
-        "        imgs = f[\"images\"]\n",
-        "        da_imgs = da.from_array(imgs, chunks=(block_frames,) + imgs.shape[1:])\n",
+        "    f1 = zarr.open_group(data_basename + postfix_f_f0 + zarr_ext, \"r\")\n",
         "\n",
+        "    imgs = f1[\"images\"]\n",
+        "    da_imgs = da.from_array(imgs, chunks=(block_frames,) + imgs.shape[1:])\n",
         "\n",
-        "        da_imgs_min, da_imgs_max = da_imgs.min(), da_imgs.max()\n",
+        "    da_imgs_min, da_imgs_max = da_imgs.min(), da_imgs.max()\n",
         "\n",
-        "        status = client.compute([da_imgs_min, da_imgs_max], lock=False, compute=False)\n",
-        "        dask.distributed.progress(status, notebook=False)\n",
-        "        print(\"\")\n",
+        "    status = client.compute([da_imgs_min, da_imgs_max], lock=False, compute=False)\n",
+        "    dask.distributed.progress(status, notebook=False)\n",
+        "    print(\"\")\n",
         "\n",
-        "        imgs_min, imgs_max = [s.result() for s in status]\n",
+        "    imgs_min, imgs_max = [s.result() for s in status]\n",
         "\n",
         "    mplsv = plt.figure(FigureClass=MPLViewer)\n",
         "    mplsv.set_images(\n",
-        "        LazyZarrDataset(fn, \"images\"),\n",
+        "        imgs,\n",
         "        vmin=imgs_min,\n",
         "        vmax=imgs_max\n",
         "    )\n",
         "\n",
-        "    lblimg = LazyZarrDataset(data_basename + postfix_rois + zarr_ext, \"labels\")\n",
-        "\n",
-        "    lblimg = lblimg[...][...]\n",
+        "    f2 = zarr.open_group(data_basename + postfix_rois + zarr_ext, \"r\")\n",
+        "    lblimg = f2[\"labels\"]\n",
         "    lblimg_msk = numpy.ma.masked_array(lblimg, mask=(lblimg==0))\n",
```

Comment:
Appears without loading the data the `mask` evaluates to `False`. To fix this, we need to load the NumPy array associated with the label image.

Reply:
Fixing with PR ( https://github.com/nanshe-org/nanshe_workflow/pull/169 ).